packages/puppeteer-core/src/cdp/Accessibility.ts

```diff
@@ -426,7 +424,7 @@ class AXNode {
     if (this.#hasFocusableChild()) {
       return false;
     }
-    if (this.#focusable && this.#name) {
+    if (this.#focusable && this.#name && this.#name !== 'Document') {
```

Comment:
Let's land this like this. We can experiment later if removing this will give as more info that we need later.